www/pages/blog.tsx

```diff
@@ -168,29 +169,31 @@ function FeaturedThumb(blog: PostTypes) {
               <p className="p text-xl">{blog.description}</p>
             </div>
 
-            {author.map((author: any) => {
-              return (
-                <div className="flex space-x-3 items-center">
-                  {author.author_image_url && (
-                    <div className="relative overflow-auto w-10 h-10">
-                      <Image
-                        src={author.author_image_url}
-                        className="rounded-full"
-                        layout="fill"
-                        objectFit="cover"
-                      />
+            <div className="grid grid-rows-4 grid-flow-col gap-4 w-max">
+              {author.map((author: any) => {
+                return (
+                  <div className="flex space-x-3 items-center">
+                    {author.author_image_url && (
+                      <div className="relative overflow-auto w-10 h-10">
+                        <Image
```

Comment:
A potential improvement could be to add ALT text to the image Something along the lines of "Author [Person's Name]".